twitter_ads/resource.py

```diff
@@ -297,7 +297,7 @@ def async_stats_job_result(klass, account, job_ids=None, **kwargs):
         resource = klass.RESOURCE_ASYNC.format(account_id=account.id)
         request = Request(account.client, 'get', resource, params=params)
 
-        return Cursor(klass, request, init_with=[account])
+        return Cursor(None, request, init_with=[account])
```

Comment:
Otherwise, this behaves differently when this method is called directly from Analytics class `Analytics. async_stats_job_result()` vs called through inherited class `Campaign.async_stats_job_result()`. The later one returns `campaign` instance and can't access return values as the Campaign class obviously doesn't have resource property for the Analytics endpoint.

https://github.com/twitterdev/twitter-python-ads-sdk/blob/master/twitter_ads/cursor.py#L96-L102